src/verify.c

```diff
@@ -474,6 +478,30 @@ static void add_missing_files(struct manifest *official_manifest)
 		if ((ret != 0) || hash_needs_work(file, local.hash)) {
 			file_not_replaced_count++;
 			printf("\n\tnot fixed\n");
+
+			if (freespace_warning_flag && cmdline_option_install) {
+				fs_free = get_available_space(path_prefix);
+				string_or_die(&original, "%s/staged/%s", state_dir, file->hash);
+				if (stat(original, &st) == 0) {
+					if (((st.st_size * 1.1) > fs_free) || fs_free < 0) {
```

Comment:
I'd expect fs_free to be zero (or some number close to zero) when there's no disk space left in this point.

Reply:
I fixed the extra unnecessary checks I was performing here. fs_free could be -1 if we we're unable to determine the free space on disk, but I think there's no need for the OR here.